newsroom/wire/block_media/permission_media.py

```diff
@@ -0,0 +1,29 @@
+from newsroom.auth import get_user
+from newsroom.companies import get_user_company
+from newsroom.products.products import get_products_by_company
+from flask import request
+
+
+class PermissionMedia:
+    @staticmethod
+    def permission_editor_in_item(item):
```

Comment:
Why use a class just to contain a method, as there is no state or other grouped functionality.

Reply:
update ,